### PR TITLE
[GITHUB-APPS] Roles and permissions guides

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
* Move content from GitLab integration page to security and permissions section
* Create how-to guide for managing org and project roles and permissions

# Reasons
Standalone project support

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/gitlab-integration.adoc [hunk 1]
```diff
@@ -250,26 +248,7 @@ To manage settings at the organization level, click btn:[Organization Settings]
...
-. Click the **Invite** button.
-
-. Enter the email address of the user you wish to invite, and select the appropriate role. You may enter multiple addresses at once, if you wish to assign these users the same role.
-+
-Organization administrator as well as organization contributor roles are currently available. Project-specific roles will be coming soon. For more information, refer to the <<#about-roles-and-permissions,Roles and permissions>> section.
-
-. An invited user will receive an email notification (sent from `noreply@circleci.com`), containing a link to accept the invite.
-+
-If they do not currently have a CircleCI account, they will need to sign up. If they already have a CircleCI account, they are added to the organization, and if they are logged in, they will see the organization as an option in the organization switcher in the top left corner of the web app.
+Add or remove users, and manage user roles for the organization as well as user invites. See the xref:roles-an-permissions-overview#[Roles and permissions overview] page for full details.
```
Comment (kbatuigas) [inline jekyll/_cci2/gitlab-integration.adoc:251]:
```suggestion
Add or remove users, and manage user roles for the organization as well as user invites. See the xref:roles-and-permissions-overview#[Roles and permissions overview] page for full details.
```

File: jekyll/_cci2/manage-roles-and-permissions.adoc [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+:page-description: How-to guides for managing roles and permissions in CircleCI.
+:icons: font
+:experimental:
+
+NOTE: If you authenticated your CircleCI account with xref:gitlab-integration#[GitLab] or xref:github-apps-integration#[GitHub apps], the content on this page applies to you.
+
+Manage user access to organizations and projects with CircleCI roles and associated permissions. The guides on this page show how to add people to your organization, update permissions, and assign specific project roles to people in your organization. For an overview of roles and permissions in CircleCI, see the xref:roles-and-permissions-overview#[Roles and permissions overview] page.
+
+[#prerquisites]
+== Prerquisites
```
Comment (kbatuigas) [inline jekyll/_cci2/manage-roles-and-permissions.adoc:18]:
```suggestion
== Prerequisites
```

File: jekyll/_cci2/manage-roles-and-permissions.adoc [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+:page-liquid:
+:page-description: How-to guides for managing roles and permissions in CircleCI.
+:icons: font
+:experimental:
+
+NOTE: If you authenticated your CircleCI account with xref:gitlab-integration#[GitLab] or xref:github-apps-integration#[GitHub apps], the content on this page applies to you.
+
+Manage user access to organizations and projects with CircleCI roles and associated permissions. The guides on this page show how to add people to your organization, update permissions, and assign specific project roles to people in your organization. For an overview of roles and permissions in CircleCI, see the xref:roles-and-permissions-overview#[Roles and permissions overview] page.
+
+[#prerquisites]
```
Comment (kbatuigas) [inline jekyll/_cci2/manage-roles-and-permissions.adoc:17]:
```suggestion
[#prerequisites]
```

File: jekyll/_cci2/manage-roles-and-permissions.adoc [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+[#add-people-to-your-organization]
+== Add people to your organization
+
+NOTE: To do this, you must have the _Organization Admin_ role.
+
+. In the CircleCI web app, click on **Organization Settings** in the sidebar
+. Click on **People**
+. Click btn:[Invite]
+. Enter the person's email address. You can add multiple people separated by a space too
+. Select a role from the dropdown menu to be applied to the people you're inviting
```
Comment (kbatuigas) [inline jekyll/_cci2/manage-roles-and-permissions.adoc:32]:
```suggestion
. Select a role from the dropdown menu to be applied to the people you are inviting
```

File: jekyll/_cci2/manage-roles-and-permissions.adoc [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+== Assign a project role
+
+image::{{site.baseurl}}/assets/img/docs/gl-ga/gitlab-project-settings-project-roles.png[Project roles setup page]
+
+. In the CircleCI web app, click on **Dashboard** in the sidebar
+. Select your project using the filter dropdown menu
+. Click btn:[Project Settings]
+. Click **People** in the sidebar menu
+. Click btn:[Assign Project Roles]
+. Select the username of the Person you want to assign a role to
```
Comment (kbatuigas) [inline jekyll/_cci2/manage-roles-and-permissions.adoc:71]:
```suggestion
. Select the username of the person you want to assign a role to
```

File: jekyll/_cci2/roles-and-permissions-overview.adoc [hunk 6]
```diff
@@ -0,0 +1,443 @@
...
+|===
+
+[#permissions-scope]
+== Permissions scope
+
+Your CircleCI roles and associated permissions are **not** derived from the permissions set in your version control system (VCS). Your CircleCI role permissions do **not** allow you to bypass permissions in the VCS.
+
+For example, you may be an _Organization Administrator_ within CircleCI, which gives you access to view and modify organization and project settings _within your CircleCI organization_. However, you will not be able to edit a project’s `.circleci/config.yml` hosted in your VCS without your user also having the write permissions _within that VCS's repository project_. Your CircleCI user’s VCS permissions are determined by its associated VCS identity.
+
+At this time, your VCS identity can be managed through your CircleCI connection when managing triggers and configurations.
```
Comment (kbatuigas) [inline jekyll/_cci2/roles-and-permissions-overview.adoc:438]: Not sure how to do this or what "VCS identity can be managed" means here exactly.

**Review by kbatuigas — CHANGES_REQUESTED** [review]
Mostly typos, one clarifying question

File: jekyll/_cci2/roles-and-permissions-overview.adoc (hunk 6 above)
Comment (rosieyohannan) [inline jekyll/_cci2/roles-and-permissions-overview.adoc:438]:
```suggestion
```

**Review by kbatuigas — APPROVED** [review]
🚀

**Pull request merged**